--- a/main/configure.in.orig	2015-01-10 19:56:35.635483317 +0100
+++ b/main/configure.in	2015-01-10 19:56:58.867482231 +0100
@@ -17,14 +17,14 @@
 COMPAT_ADDIN_VERSION=m4_esyscmd_s([grep ^CompatVersion ../version.config|cut -d "=" -f 2|tr -d '\n'])
 
 AC_PATH_PROG(MONO, mono)
-AC_PATH_PROG(MCS, gmcs)
+AC_PATH_PROG(MCS, mcs)
 
 if test "x$MONO" = "x" ; then
 	AC_MSG_ERROR([Can't find "mono" in your PATH])
 fi
 
 if test "x$MCS" = "x" ; then
-  AC_MSG_ERROR([Can't find "gmcs" in your PATH])
+  AC_MSG_ERROR([Can't find "mcs" in your PATH])
 fi
 
 AC_PATH_PROG(PKG_CONFIG, pkg-config, no)

